### PR TITLE
win32: Added Powershell way, WMIC still works

### PR DESCRIPTION
**Pull request by erickriva**

### CODE REVIEW

File: win32.js [hunk 1]
```diff
@@ -16,15 +19,20 @@ const spawnOpts = {
...
 
   for (let line of (gwTable || "").trim().split(/\r?\n/).splice(1)) {
     line = line.trim();
     const [_, gwArr, gwCostsArr, id, ipMetric] = /({.+?}) +?({.+?}) +?([0-9]+) +?([0-9]+)/g.exec(line) || [];
     if (!gwArr) continue;
 
-    const gateways = (gwArr.match(/"(.+?)"/g) || []).map(match => match.substring(1, match.length - 1));
+    let finalGwArr = gwArr;
+    if (command === "powershell") {
+      finalGwArr = `{ ${gwArr.replace(/[{}]/g, "").split(", ").map((value) => `"${value}"`).join(", ")} }`;
```
Comment (erickriva) [inline win32.js:32]: Powershell table doesn't show IP's between `" "`. Added it for powershell only. I can't find a better way.

File: win32.js [hunk 2]
```diff
@@ -48,8 +48,8 @@ function parseGwTable(gwTable, family, command) {
   if (bestGw) return [bestGw, bestId];
 }
 
-function parseIfTable(ifTable) {
-  const line = (ifTable || "").trim().split("\n")[1];
+function parseIfTable(ifTable, command) {
+  const line = (ifTable || "").trim().split("\n")[command === "powershell" ? 2 : 1];
```
Comment (erickriva) [inline win32.js:52]: Powershell table 2nd line is a separator, like this: `----------`

**Comment by erickriva** [comment]
image

**Comment by openspeedtest** [comment]
@silverwind Could you please test and merge this? Thank you @erickriva

**Pull request closed without merging**